packages/block-editor/src/components/block-settings/container.native.js

```diff
@@ -6,12 +6,18 @@ import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
 import { InspectorControls } from '@wordpress/block-editor';
 
+/**
+ * Internal dependencies
+ */
+import styles from './container.native.scss';
+
 function BottomSheetSettings( { editorSidebarOpened, closeGeneralSidebar, ...props } ) {
 	return (
 		<BottomSheet
 			isVisible={ editorSidebarOpened }
 			onClose={ closeGeneralSidebar }
 			hideHeader
+			contentStyle={ styles.content }
```

Comment:
> I'm wondering if this zero-padding should be the default in BottomSheet and let the content itself add padding if needed?

I think so, if I'm understanding correctly. The space for the drag handle should always be 16px, and the content area starts below that – different types of content should inform the padding/margin from there.

Reply:
> I found some difference in padding between block settings and link settings that I don't think should be there:

You're right – that button should be full width across the board. More guidance incoming in a comment on the main thread...

Reply:
> I'm wondering if this zero-padding should be the default in BottomSheet and let the content itself add padding if needed?

In my opinion it depends how many components within `BottomSheet` has to be padded. 
In our case, this only applies to the section with two buttons, so I've decided to add new prop `padded` which controls if section wrapped in `PanelBody` needs a padding.

Reply:
> I'm wondering if this zero-padding should be the default in BottomSheet and let the content itself add padding if needed?

In our case I had to have zero-padding in `BottomSheet` to make 100% width buttons. Generally I think that it can be default in `BottomSheet`, especially that in other `BottomSheet` [usage](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/inserter/menu.native.js#L62) there is also a style which is resetting padding. It's definitely worth discussing.